kreuzberg/cli.py

```diff
@@ -0,0 +1,130 @@
+import sys
+from pathlib import Path
+from typing import Optional, TextIO
+
+import click
+import tomli
+
+from kreuzberg import ExtractionConfig, ExtractionResult, extract_bytes, extract_file
+from kreuzberg._types import TesseractConfig
+
+
+def load_config(config_path: Optional[Path] = None) -> dict:
+    """Load configuration from pyproject.toml."""
+    if config_path:
+        toml_path = config_path
+    else:
+        toml_path = Path("pyproject.toml")
+
+    if not toml_path.exists():
+        return {}
+
+    with open(toml_path, "rb") as f:
```

Comment:
Use `Path(...).read_text` or read_pytes< don't use the legacy context manager open